src/Discord.Net.WebSocket/Entities/Channels/SocketDMChannel.cs

```diff
@@ -51,13 +54,30 @@ public Task CloseAsync(RequestOptions options = null)
         /// <inheritdoc />
         public SocketMessage GetCachedMessage(ulong id)
             => _messages?.Get(id);
+        /// <summary>
+        ///     Gets the message associated with the passed <paramref name="id"/>.
+        /// </summary>
+        /// <param name="id">The snowflake identifier of the message you want to retrieve.</param>
+        /// <param name="options">The options to be used when sending the request.</param>
+        /// <returns>
+        ///     An awaitable <see cref="Task"/> containing an <see cref="IMesage"/>.
+        /// </returns>
         public async Task<IMessage> GetMessageAsync(ulong id, RequestOptions options = null)
         {
             IMessage msg = _messages?.Get(id);
             if (msg == null)
                 msg = await ChannelHelper.GetMessageAsync(this, Discord, id, options).ConfigureAwait(false);
             return msg;
         }
+
+        /// <summary>
+        ///     Gets a nested collection of messages.
+        /// </summary>
+        /// <param name="limit">The number of messages you want to get.</param>
+        /// <param name="options">The options to be used when sending the request.</param>
+        /// <returns>
+        ///     An awaitable <see cref="Task"/> containing an <see cref="IReadOnlyCollection{IMessage}"/>.
```

Comment:
It is not an awaitable Task, and you should refer to the interface impl for the intended XML doc